api/src/main/java/io/strimzi/api/kafka/model/MaintenanceSpec.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class MaintenanceSpec implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private String schedule;
```

Comment:
* Is one field enough? Don't we need a start and end?
* Is the cron format able to describe different windows for different days in single string? Or should we rather use a List here to allow users to for example specify that the maintenance windows are Monday Mon-Fri 0:00-5:00 + Mon-Fri 23:00-24:00 + Sat-Sun 00:00-24:00?

Reply:
in the beginning, I had a start + end but then I realized that, as in your examples, a cron expression can contain interval in hours and days (i.e. Mon-Fri 0:00-5:00). 
The problem is about having more of them, in this case, a single field isn't enough and a List is needed.

Reply:
Ok, if the Cron expression can show both, it sounds like single String is good enough ... but the List might be still handy.

Reply:
Yeah I agree that a List is useful.